dns-policy-operator/src/database.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Provide the DatabaseObserver class to handle database relation and state."""
```

Comment:
This is not a `DatabaseObserver` but a `DatabaseHandler`

Reply:
Fixed